.github/workflows/manylinux_2014.yml

```diff
@@ -188,4 +207,30 @@ jobs:
         with:
           name: openvino_wheels
           path: ${{ env.INSTALL_WHEELS_DIR }}/wheels/*.whl
-          if-no-files-found: 'error'
\ No newline at end of file
+          if-no-files-found: 'error'
+          
+      - name: Store artifacts to a shared drive
+        id: store_artifacts
+        if: ${{ always() }}
+        uses: ./src/.github/actions/store_artifacts
+        with:
+          artifacts: |
+            ${{ env.BUILD_DIR }}/openvino_package.tar.gz
+            ${{ env.MANIFEST_PATH }}
+            ${{ env.INSTALL_WHEELS_DIR }}
```

Comment:
I see `wheels/wheels/*.whl` on share, so either need to fix the path itself or provide  `${{ env.INSTALL_WHEELS_DIR }}/wheels` here

Reply:
fixed